samples/sample_common/include/sample_utils.h

```diff
@@ -110,6 +111,106 @@ enum ExtBRCType {
     EXTBRC_IMPLICIT
 };
 
+namespace QPFile {
+#   ifdef max
```

Comment:
why do we need it? maybe you need to add ```NOMINMAX``` to proj file?

Reply:
It seems NOMINMAX have been already added (or where can i add it?). If i remove #undef, max macro overlaps std::numeric_limits::max.